### PR TITLE
(non-standard-operations) add tier2,3 and 4 to computer shutdown 

### PR DESCRIPTION
**Pull request by csilva-cl**

### CODE REVIEW

File: Non-Standard-Operations/Computer-Room-Shutdown/index.rst [hunk 1]
```diff
@@ -63,4 +70,56 @@ auxtel-dc01.cp.lsst.org - can always be shut down first, it is only used for ima
...
+auxtel-daq-mgt.cp.lsst.org - can always be shut down if we are not taking data, will lose connection/monitoring of the WREB a
+
+
+Tier 2
+^^^^^^
+The following is the list computers will be powered off alerting the system owners, but not taking down the control system
+
+* Control System
+yagan[13-20].cp.lsst.org
+azar[02-03].cp.lsst.org
```
Comment (edennihy) [inline Non-Standard-Operations/Computer-Room-Shutdown/index.rst]: I think we now have control system dependencies to the /net/obs nfs mount on azar02 via the scriptqueue and scheduler. If so, we should move azar02 to Tier 3 (but not azar03). @tribeiro can you confirm that is the case?

**Review by edennihy — COMMENTED** [review]
One minor comment, I have asked Tiago Ribeiro for confirmation. Otherwise this looks good to me. Once we get the confirmation from Tiago I will approve.

**Comment by bstalder** [comment]
There should be another tier between 3 and 4, which is supporting camera liveness (does not need the control network, but does require the camera clusters).  this includes the switches, DAQs, and most CCS machines.

**Comment by csilva-cl** [comment]
The tiers in addition to an order of shutdown, represent a threat level for the hardware, so adding another tier wouldn't change the end result since tier 3 is already a high threat level that requires to shutdown as much as possible (including the daqs). Tier 3 also requires coordination with "system owners" so that would be sufficient to wait for the warm-ups.

**Review by tribeiro — APPROVED** [review]
Looks good to me!

File: Non-Standard-Operations/Computer-Room-Shutdown/index.rst (hunk 1 above)
Comment (tribeiro) [reply Non-Standard-Operations/Computer-Room-Shutdown/index.rst]: I think at this point we can consider that we are running the system in degraded mode and won't be able to update the obs env. 

Alternatively, we can move this functionality to azar1. I don't think it is worth keeping an additional node for this.

**Pull request merged**